src/server/requestHandlers.js

```diff
@@ -111,50 +109,40 @@ fluid.defaults("sjrk.storyTelling.server.getStoryHandler", {
  *
  * @param {Object} request - a Kettle request that includes an ID for the story to retrieve
  * @param {Component} dataSource - an instance of sjrk.storyTelling.server.dataSource.couch.story
- * @param {String} uploadedFilesHandlerPath - the path on the server to uploaded files
  */
-sjrk.storyTelling.server.handleGetStory = function (request, dataSource, uploadedFilesHandlerPath) {
+sjrk.storyTelling.server.handleGetStory = function (request, dataSource) {
     var id = request.req.params.id;
     var promise = dataSource.get({directStoryId: id});
 
-    promise.then(function (response) {
+    var noAccessErrorMessage = "An error occurred while retrieving the requested story";
 
-        fluid.transform(response.content, function (block) {
-            if (block.blockType === "image") {
-                if (block.imageUrl) {
-                    block.imageUrl = uploadedFilesHandlerPath + "/" + block.imageUrl;
-                }
-                return block;
-            } else if (block.blockType === "audio" || block.blockType === "video") {
-                if (block.mediaUrl) {
-                    block.mediaUrl = uploadedFilesHandlerPath + "/" + block.mediaUrl;
-                }
-                return block;
-            }
-        });
+    promise.then(function (response) {
+        if (response.published) {
+            request.events.onSuccess.fire(JSON.stringify(response));
+        } else {
+            fluid.log("Unauthorized: cannot access an unpublished story: " + id);
 
-        var responseAsJSON = JSON.stringify(response);
-        request.events.onSuccess.fire(responseAsJSON);
+            request.events.onError.fire({
+                isError: true,
+                message: noAccessErrorMessage
+            });
+        }
     }, function (error) {
-        var errorAsJSON = JSON.stringify(error);
+        fluid.log("Error getting story with ID " + id + ", error detail: " + JSON.stringify(error));
```

Comment:
This seems also like it should have a log level of Warn

Reply:
Warn seems reasonable here, though if we get a large number of attempts to access stories without authorization, then we may want to drop this to "Important" if the logs are getting too big

Reply:
We should probably sort out log backups first ;P